### PR TITLE
Backport v4 to Cake3

### PR DESCRIPTION
**Pull request by magic-77**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
 		}
 	],
 	"require": {
-		"php": ">=5.6",
+		"php": ">=7.4",
```
Comment (dereuromark) [inline composer.json:16]: Any reason to go higher with a legacy branch than the current master?
https://github.com/dereuromark/cakephp-captcha/blob/master/composer.json#L21

File: composer.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
-	"name": "dereuromark/cakephp-captcha",
+	"name": "magic-77/cakephp-captcha",
 	"type": "cakephp-plugin",
 	"description": "A CakePHP plugin to integrate captchas.",
 	"keywords": ["cakephp", "plugin", "captchas", "security"],
-	"homepage": "https://github.com/dereuromark/cakephp-captcha",
+	"homepage": "https://github.com/magic-77/cakephp-captcha",
```
Comment (dereuromark) [inline composer.json:6]: These need to be reverted

Comment (magic-77) [reply composer.json:6]: I don't see a revert Button, i only can edit the File. Will be this ok for you?

**Comment by dereuromark** [comment]
You might also want to backport https://github.com/dereuromark/cakephp-captcha/commit/84390fdb0518f0fb28c0bab970bd7a792d8a4422

**Comment by magic-77** [comment]
i think the easiest way would be to close the pull request. agree with that.
please excuse the circumstances

**Comment by dereuromark** [comment]
If you are not interested anymore in those changes being backported to 3.x, sure.

**Pull request closed without merging**